src/modules/groups/infrastructure/repositories/group.repository.ts

```
@@ -0,0 +1,113 @@
+    const query = this.repository
+      .createQueryBuilder('group')
+      .where('group.id = :id', { id })
+      .leftJoin('group.servers', 'server')
+      .leftJoin('group.vms', 'vm')
+      .select('group')
+      .addSelect('COUNT(DISTINCT server.id)', 'serverCount')
+      .addSelect('COUNT(DISTINCT vm.id)', 'vmCount')
+      .groupBy('group.id');
+
+    const result = await query.getRawAndEntities();
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)